tests/unit/CompleteResolution.spec.ts

```diff
@@ -63,57 +63,42 @@ for (const test of completeResolutionTestCases) {
   describe(`Complete Resolution view for a ${test.entityType}`, () => {
     let wrapper: any
 
-    it('renders the complete resolution component', async () => {
+    beforeAll(() => {
       wrapper = wrapperFactory(
         CompleteResolution,
         null,
-        { entityType: test.entityType },
+        {
+          entityType: test.entityType,
+          business: { foundingDate: '2021-10-07T20:37:41+00:00' },
+          createResolutionStep: {
+            givenName: 'Given',
+            additionalName: 'Additional',
+            familyName: 'Family'
+          }
+        },
         null,
         DissolutionResources
       )
+    })
 
+    it('renders the complete resolution component', () => {
       expect(wrapper.find('#complete-resolution').exists()).toBe(true)
     })
 
-    it('renders intro section', async () => {
-      wrapper = wrapperFactory(
-        CompleteResolution,
-        null,
-        { entityType: test.entityType },
-        null,
-        DissolutionResources
-      )
```

Comment:
These were all the same so I made them common.

Also, most tests here were'e async so I updated that.